locale/en/get-involved/node-events.md

```diff
@@ -10,11 +10,15 @@ This is a list of Node.js events and conferences. Please submit a PR if you'd li
 ## Notes for adding events
 
 FORMAT
-- [Event Name](https://) - Location, Dates
+
+- [Event Name](https://) | Location | Dates
 
 ## Events
 
 ### 2019
-- [NodeConf Colombia](https://colombia.nodeconf.com) - Medellin, Colombia, June 21-22 
-- [NodeConf EU](https://www.nodeconf.eu/2019.html) - Lyrath Estate, Kilkenny, November 10-13
-- [NodeSummit](https://www.nodesummit.com) - San Francisco, TBA 
+
```

Comment:
For 'node-speakers' and 'node-meetups', we don't use table style. So do we really need to use the table style, or keep them in union?

Reply:
So perhaps update node-speakers and node-meetups to use table style? 😜

I like the table style more and it looks a lot more readable.
Perhaps we can try both styles and see what works better. I actually think that there would be more speakers and more meetups than conferences so it makes even more sense to really consider updating them to table style.

Reply:
@lirantal：Agree with you :)